### DISCUSSION TITLE
Didn't work low-level Router in SPA mode

### DISCUSSION THREAD

**Discussion by kaufd** [jaredpalmer/razzle#1709] [Q&A] [answered]
When I use  BrowserRouter from 'react-router-dom' all is work fine. But when I try to use low-level Router from 'react-router' pages don't render and nothing happens with no errors. 
What could be the reason for that?

**Comment by fivethreeo** [comment, marked as answer]
You should use browserrouter in spa mode.

**Reply by kaufd** [reply]
It's work with Router also. The problem was in wrong version of history packet (it was 5th not 4th)
